FreeTubeApp/FreeTube
# Upgrade electron from 22.x to 26.x and replace deprecated `protocol.registerBufferProtocol`

## Pull Request Type


## Related issue



Closes https://github.com/FreeTubeApp/FreeTube/pull/3960

## Description

https://github.com/FreeTubeApp/FreeTube/pull/3960 plus replace deprecated `protocol.registerBufferProtocol`

## Screenshots 

N/A

## Testing 



- Enable `Replace HTTP Cache`
- See https://github.com/FreeTubeApp/FreeTube/pull/2498 & https://github.com/FreeTubeApp/FreeTube/pull/2902

## Desktop

- **OS:**
- **OS Version:**
- **FreeTube version:**

## Additional context
